.travis.yml

```diff
@@ -195,6 +194,7 @@ install:
   - git submodule --quiet update --init libs/ratio
   - git submodule --quiet update --init libs/rational
   - git submodule --quiet update --init libs/regex
+  - git submodule --quiet update --init libs/smart_ptr
```

Comment:
why did you add it here ? I thought we no longer needed it ?

Reply:
It moved from group of direct deps to transitive deps. - it still is required by some other Boost libs.

Reply:
I see.

Reply:
Comment where? commit log, .travis.yml

The .travis.yml already has comments indicating if a library belongs to direct or transitive dependencies:

https://github.com/mloskot/gil/blob/0eae37af54b6de79dc4dc0d0255b76154daef66f/.travis.yml#L156-L205

https://github.com/mloskot/gil/blob/0eae37af54b6de79dc4dc0d0255b76154daef66f/.travis.yml#L156-L205

Reply:
OK, fair enough.